appsflyer/src/main/java/com/tealium/remotecommands/appsflyer/AppsFlyerInstance.kt

```diff
@@ -28,16 +28,16 @@ class AppsFlyerInstance(
         configSettings: Map<String, Any>?
     ) {
         configSettings?.let { settings ->
-            if (settings.containsKey(Config.MIN_TIME_BETWEEN_SESSIONS)) {
-                setMinsBetweenSessions(settings[Config.MIN_TIME_BETWEEN_SESSIONS] as Int)
+            if (settings.containsKey(Settings.TIME_BETWEEN_SESSIONS)) {
```

Comment:
Also unrelated to this PR, but let's update the `getApplication` method name to actually describe what it's doing. Given that it is not returning the `Application` object, but rather registering the application lifecycle listener.